src/modules/views/Sharings/index.jsx

```diff
@@ -84,6 +88,22 @@ export const SharingsView = ({ sharedDocumentIds = [] }) => {
   )
   const result = useQuery(query.definition, query.options)
 
+  const filteredResult = useMemo(
+    () => ({
+      ...result,
+      data:
+        tab === 1
+          ? result.data.filter(item => item.dir_id === SHARED_DRIVES_DIR_ID)
+          : result.data,
+      count:
+        tab === 1
```

Comment:
Can you find a way to make the condition more explicit, so avoiding `tab === 1` but something like `TAB === SHARED_DRIVE_TAB` or `isShareDriveTab(tab)`. Same for `const [tab, setTab] = useState(0)`.